src/FasterImage/FasterImage.php

```diff
@@ -68,6 +68,23 @@ class FasterImage
      */
     public function batch(array $urls)
     {
+        $has_curl_multi = (
+            function_exists( 'curl_multi_init' )
+            &&
+            function_exists( 'curl_multi_exec' )
+            &&
+            function_exists( 'curl_multi_add_handle' )
+            &&
+            function_exists( 'curl_multi_select' )
+            &&
+            defined( 'CURLM_OK' )
+            &&
+            defined( 'CURLM_CALL_MULTI_PERFORM' )
+        );
+        $has_curl_multi = false; // TEMP for unit testing.
```

Comment:
Remove this line once tests are confirmed to pass:

```suggestion
```

I wasn't sure how best to configure the test environment for cURL multi not being available.

Reply:
I think the best way to unit test this will unfortunately be to mock the `function_exists` and `defined` functions to return false using something like https://github.com/php-mock/php-mock. It should be pretty easy then to trigger the single curl batch path.


```php
$builder = new MockBuilder();
$builder->setNamespace(__NAMESPACE__)
        ->setName("function_exists")
        ->setFunction(
            function () {
                return false;
            }
        );
                    
$mock = $builder->build();

```